MarvelApiClientTests/ThumbnailDTOTests.swift

```diff
@@ -19,7 +19,7 @@ class ThumbnailDTOTests: XCTestCase {
     func testGeneratesFullSizeThumbnailURLProperly() {
         let thumbnail = givenAThumbnailDTO(anyPath, anyFormat)
 
-        let thumbnailURL = thumbnail.URL(.FullSize)
+        let thumbnailURL = thumbnail.URL(variant: .FullSize)
```

Comment:
Check the ThumbnailDTOVariant, Swift 3 conventions says enum values should start in lowercase aka `.fullSize`